internal/engine/sqlite/convert.go

```diff
@@ -168,6 +168,12 @@ func (c *cc) convertCreate_virtual_table_fts5(n *parser.Create_virtual_table_stm
 		}
 	}
 
+	stmt.Cols = append(stmt.Cols, &ast.ColumnDef{
+		Colname: identifier(stmt.Name.Name),
+		IsNotNull: true,
+		TypeName: &ast.TypeName{Name: "text"},
```

Comment:
There's a problem with doing it like this: if you `SELECT <table_name> FROM <table_name>`, you get an `integer`, but when you use `MATCH`, you are expected to provide a `string`.